doc/changes/changes_3.3.1.md

```diff
@@ -5,6 +5,7 @@ This release updates the poetry lock file. Especially, the Exasol Python Toolbox
 ## Refactorings
 
  - #161: Migrated to PTB 1.7.3
+ - #164: Migrated to PTB 1.7.4
```

Comment:
3.3.1 was already released, I think. I'm not really familiar with the SLC code, so I'll defer to @tomuben there regarding if another patch release is needed for this or not.
```suggestion

```